blocks/svg/svg.js

```diff
@@ -0,0 +1,28 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+
+export default async function decorate(block) {
+  const config = readBlockConfig(block);
+  const caption = document.createElement('div');
+  caption.innerHTML = config.caption || '';
+
+  const keys = Object.keys(config);
+
+  const [ratioKey] = keys.filter((k) => k.match(/(aspect|ratio)/));
+  let ratio = '1:1';
+  if (ratioKey) {
+    ratio = config[ratioKey];
```

Comment:
```suggestion
    ratio = config[ratioKey]?.trim();
```
protect against easy to add leading/trailing whitespace?